service/src/main/java/dk/kvalitetsit/stakit/service/GroupServiceImpl.java

```diff
@@ -89,4 +92,29 @@ public boolean patchGroup(UUID groupUuid, List<UUID> serviceList) {
         }
         return successGroup;
     }
+
+    @Override
+    @Transactional
+    public Optional<List<ServiceModel>> getServicesInGroup(UUID uuid) {
+        if (groupConfigurationDao.findByUuid(uuid).isEmpty()) {
+            return Optional.empty();
+        }
+        var dbResult = serviceConfigurationDao.findByGroupUuid(uuid);
+        List<ServiceModel> serviceList = new ArrayList<>();
+
+        for (ServiceConfigurationEntityWithGroupUuid service : dbResult) {
+            var model = new ServiceModel(service.name(), service.service(), service.ignoreServiceName(), uuid, service.uuid(), service.description());
+            serviceList.add(model);
+        }
+
+        return Optional.of(serviceList);
+    }
```

Comment:
Her kan du også anvende en stream hvis du hellere vil det. 

`return Optional.of(dbResult.stream().map(x -> new ServiceModel(x.name(), x.service(), x.ignoreServiceName(), uuid, x.uuid(), x.description())).toList());` 

Bør kunne ersatte linie 103 til 110. 

Den nuværende løsning virker fint og det er op til dig hvad du synes bedst om.